CS/helper/com/vehicledekho/helper/JdbcSaveBike.java

```diff
@@ -0,0 +1,50 @@
+package com.vehicledekho.helper;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+import com.vehicledekho.jdbc.Connection;
+import com.vehicledekho.model.Bike;
+import com.vehicledekho.model.Vehicle;
+import com.vehicledekho.model.VehicleType;
+
+public class JdbcSaveBike {
+	public static void writeToDatabase(Vehicle vehicle) {
+		PreparedStatement preparedStatement = null;
+		java.sql.Connection connection = Connection
+				.getConnection();
+		String writeQueryForBike = "insert into VEHICLE (TYPE,ENGINE_IN_CC,FUEL_CAPACITY,MILEAGE,PRICE,ROAD_TAX,SELF_START,HELMET_PRICE) values (?,?,?,?,?,?,?,?)";
+		try {
+			// setting up prepared statement
+			if (vehicle instanceof Bike) {
+				preparedStatement = connection
```

Comment:
There is a scenario when you get null Connection. Have you checked this. First check the null and then try to access the connection to get rid the problem of NullPointerException.